### PR TITLE
Add total borrows, borrow cap & borrow paused status to Compound resolver

### PR DESCRIPTION
**Pull request by mubaris**

### CODE REVIEW

File: contracts/protocols/compound.sol [hunk 1]
```diff
@@ -132,20 +138,24 @@ contract Resolver is Helpers {
...
                 cToken.balanceOf(owner),
                 cToken.borrowBalanceStored(owner),
+                cToken.totalBorrows(),
+                troller.borrowCaps(cAddress[i]),
                 cToken.supplyRatePerBlock(),
                 cToken.borrowRatePerBlock(),
                 collateralFactor,
-                isComped
+                isComped,
+                troller.borrowGuardianPaused(cAddress[i])
```
Comment (thrilok209) [inline contracts/protocols/compound.sol:158]: @mubaris What's the use of `borrowGuardianPaused`? If borrowing allowed or not? Or something else?

Comment (mubaris) [reply contracts/protocols/compound.sol:158]: yes, to check if borrow is paused or not

https://github.com/compound-finance/compound-protocol/blob/c5fcc34222693ad5f547b14ed01ce719b5f4b000/contracts/Comptroller.sol#L332

Comment (thrilok209) [reply contracts/protocols/compound.sol:158]: Got it.

File: contracts/protocols/compound.sol [hunk 2]
```diff
@@ -132,20 +138,24 @@ contract Resolver is Helpers {
...
-            (,uint collateralFactor, bool isComped) = getComptroller().markets(address(cToken));
+            (,uint collateralFactor, bool isComped) = troller.markets(address(cToken));
             tokensData[i] = CompData(
                 priceInETH,
                 priceInUSD,
                 cToken.exchangeRateStored(),
                 cToken.balanceOf(owner),
                 cToken.borrowBalanceStored(owner),
+                cToken.totalBorrows(),
+                troller.borrowCaps(cAddress[i]),
```
Comment (thrilok209) [inline contracts/protocols/compound.sol:153]: @mubaris For unlimited borrowing, it's `0`, right?

**Review by thrilok209 — APPROVED** [review]
LGTM

File: contracts/protocols/compound.sol (hunk 2 above)
Comment (mubaris) [reply contracts/protocols/compound.sol:153]: Yes, 0 is unlimited

**Pull request merged**